commands/Moderation/nickname.js

```diff
@@ -0,0 +1,29 @@
+exports.run = async (client, msg, [user, name]) => {
+  // Checks to see if the nickname is too long or too short for the discord requirement if so replies and then exits.
+  if (name.length > 32 || name.length === 0) {
```

Comment:
And remove these checks as the parameter requires a string of 1-32 chars.

Reply:
Pre sure that is checking if name.length is **greater** than 32 or is 0 to make a warning that the nickname is too long or short.

Reply:
No, that'll never trigger the check if the message is 0 chars long or more than 32, check usage.